src/steps/10-FinancialDetails/IGCE/SupportingDocumentation.vue

```diff
@@ -104,12 +104,13 @@ export default class SupportingDocumentation extends Vue {
    * code that uploads the attachment.
    */
   async saveRequirementCostEstimateData(): Promise<void> {
-    const requirementsCostEstimate = await FinancialDetails.loadRequirementsCostEstimate();
-    this.loaded = await FinancialDetails.saveRequirementsCostEstimate(requirementsCostEstimate);
+    const requirementsCostEstimate = await IGCE.getRequirementsCostEstimate();
+    await IGCE.saveRequirementsCostEstimate(requirementsCostEstimate);
+    this.loaded = await IGCE.getRequirementsCostEstimate();
   }
 
   async loadRequirementsCostEstimateData(): Promise<void>{
-    this.loaded = await FinancialDetails.loadRequirementsCostEstimate();
+    this.loaded = await IGCE.getRequirementsCostEstimate();
   }
```

Comment:
`loaded` variable doesn't seem to be used meaningfully in the code

Reply:
Didn't make any functional change to that component @tonysumner-ccpo  except for pointing the code to the correct store. Should I update this component as part of this PR?